template/lambda-sqs-worker-cdk/package.json

```diff
@@ -18,13 +18,13 @@
   "devDependencies": {
     "@aws-cdk/assert": "^2.24.0",
     "@types/aws-lambda": "^8.10.82",
-    "@types/node": "16.11.64",
-    "aws-cdk": "^2.24.0",
-    "aws-cdk-lib": "^2.24.0",
+    "@types/node": "^18.11.9",
```

Comment:
Any reason we're pinning? Happy to go back to pinned.

Reply:
I think there was a breaking change in a later v16 version, but this is good given v18 latest is passing